### PR TITLE
ICDS Dashboard: barcharts in mobile

### PR DESCRIPTION
**Pull request by saiprakash-v**
https://trello.com/c/0lN5ACOz
##### SUMMARY

display barcharts in demographics page of awc reports

### CODE REVIEW

File: custom/icds_reports/static/icds_reports/mobile/css/app.less [hunk 1]
```diff
@@ -308,3 +308,13 @@
 .float-left {
   float: left;
 }
+.bar-chart {
+  .bg-white;
+  height: 415px;
+  margin-bottom: 5px;
+  overflow: auto;
+}
+svg {
```
Comment (czue) [inline custom/icds_reports/static/icds_reports/mobile/css/app.less]: this selector feels a bit aggressive. what's it doing? any way to add a class or id attribute so it doesn't affect every svg image?

File: custom/icds_reports/static/js/directives/awc-reports/awc-reports.directive.js [hunk 2]
```diff
@@ -2064,6 +2064,51 @@ function AwcReportsController($scope, $http, $location, $routeParams, $log, DTOp
         },
     };
 
+    vm.mobileChartOptions = {
```
Comment (czue) [inline custom/icds_reports/static/js/directives/awc-reports/awc-reports.directive.js]: can you refactor this so there's a `chartOptionsBase` and then we make a copy of it and override just what is different for web and mobile?

**Review by czue — APPROVED** [review]
Looks good, but let's try and do more re-use.

**Review by czue — CHANGES_REQUESTED** [review]
sorry, i actually did not mean to approve yet.

File: custom/icds_reports/static/icds_reports/mobile/css/app.less (hunk 1 above)
Comment (saiprakash-v) [reply custom/icds_reports/static/icds_reports/mobile/css/app.less]: I didnt mean this to be final pr. I will change it

**Review by czue — APPROVED** [review]

**Pull request merged**